auto_round/autoround.py

```diff
@@ -1015,7 +1045,14 @@ def quant_blocks(
                 q_input=q_input,
                 device=device,
             )
-            m = m.to("cpu")
+            if self.block_wise:
+                if isinstance(m, WrapperMultiblock):
+                    for layer in m.layers:
+                        layer = layer.to("meta")
+                else:
+                    m = m.to('meta')
+            if not self.model.device.type == "meta":
+                m = m.to("cpu")
             torch.cuda.empty_cache()
 
         del q_input
```

Comment:
where should the trained parameters to be saved, the size is typically close to the model size